src/cirrus/lib2/process_payload.py

```diff
@@ -354,11 +362,13 @@ def process(self, replace=False):
                 or payload["id"] in payload_ids["skipped"]
                 or payload["id"] in payload_ids["failed"]
             ):
-                logger.warning(f"Dropping duplicated payload {payload['id']}")
```

Comment:
Was dropping this log message intentional? It still seems relevant unless I'm mistaken.

Reply:
I had previously toyed with moving the logging into the state-change functions, but that quickly showed itself to lead to problems.  Failed to revert this line.